docs/types/mdx-js.d.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * https://mdxjs.com/advanced/typescript
+ */
+
+declare module '@mdx-js/runtime';
```

Comment:
I believe MDX 2 brings improved typescript support, let's hope this is better by then 😄